libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -190,6 +190,14 @@ fun readableMapOf(invoicePaidDetails: InvoicePaidDetails): ReadableMap {
     )
 }
 
+fun readableMapOf(paymentFailedData: PaymentFailedData): ReadableMap {
+    return readableMapOf(
+            "error" to paymentFailedData.error,
+            "bolt11" to readableMapOf(paymentFailedData.bolt11)
```

Comment:
If bolt11 is optional then `readableMapOf(lnInvoice: LnInvoice)` needs to handle the null case.

Reply:
Right, missed that. Fixed.